Slim/Control/Queries.pm

```diff
@@ -296,6 +296,7 @@ sub albumsQuery {
 	my $work	         = $request->getParam('work_id');
```

Comment:
Please remove the tab after `$work` to fix the alignment. Now that's really important.